### ISSUE TITLE
error since 2024.5 - Config flow could not be loaded: {"message":"Invalid handler specified"}

### ISSUE THREAD

**Issue by momoz** [m50/ha-fallback-conversation#10]
I've tried deleting the integration and reinstalling and continue to get this error when I try to setup:
Config flow could not be loaded: {"message":"Invalid handler specified"}

Below is from the log too.

```
2024-05-13 11:02:55.152 WARNING (MainThread) [homeassistant.util.loop] Detected blocking call to import_module inside the event loop by integration 'config' at homeassistant/components/config/config_entries.py, line 179: return await super()._post_impl(request, data) (offender: /usr/src/homeassistant/homeassistant/loader.py, line 1270: return importlib.import_module(f"{self.pkg_path}.{platform_name}")), please create a bug report at https://github.com/home-assistant/core/issues?q=is%3Aopen+is%3Aissue+label%3A%22integration%3A+config%22
2024-05-13 11:02:55.152 ERROR (MainThread) [homeassistant.config_entries] Error occurred loading flow for integration fallback_conversation: cannot import name '_get_agent_manager' from 'homeassistant.components.conversation' (/usr/src/homeassistant/homeassistant/components/conversation/__init__.py)

```

**Comment by t0bst4r** [comment]
This is related to #6 and will be fixed with #7, but unfortunately @m50 is not answering.

**Comment by m50** [comment]
Sorry about that @t0bst4r and @momoz, been taking a mental health break, which included me stepping away from personal projects as well. Realized today that it was broken.

Merged #7, which should fix this, so will close this now.
